src/common/formatHelpers/index.ts

```diff
@@ -15,13 +15,13 @@
  * @module formatHelpers
  */
 
-export { default as createPropertyFormatter } from './createPropertyFormatter.js';
-export { default as fileHeader } from './fileHeader.js';
-export { default as formattedVariables } from './formattedVariables.js';
-export { default as getTypeScriptType } from './getTypeScriptType.js';
-export { default as iconsWithPrefix } from './iconsWithPrefix.js';
-export { default as sortByReference } from './sortByReference.js';
-export { default as sortByName } from './sortByName.js';
-export { default as minifyDictionary } from './minifyDictionary.js';
-export { default as setSwiftFileProperties } from './setSwiftFileProperties.js';
-export { default as setComposeObjectProperties } from './setComposeObjectProperties.js';
+export { default as createPropertyFormatter } from './createPropertyFormatter';
```

Comment:
For future: export named functions and don't have to do `export { default as _ }`